Foody/Onboarding/Login/LoginViewController.swift

```diff
@@ -6,29 +6,112 @@
 //
 
 import UIKit
+import FirebaseAuth
+import Firebase
 
 class LoginViewController: UIViewController {
 
+    @IBOutlet weak var loginImageView: UIImageView!
+    
+    @IBOutlet weak var loginLbl: UILabel!
+    @IBOutlet weak var welcomeLbl: UILabel!
+    
+    @IBOutlet weak var rememberMeSwitch: UISwitch!
+    
+    @IBOutlet weak var emailTextFld: UITextField!
+    @IBOutlet weak var passwordTextFld: UITextField!
+    
+    @IBOutlet weak var loginBttn: UIButton!
+    @IBOutlet weak var registerBttn: UIButton!
+    
     var clickedRegisterCompletion:(() -> ())?
     
     override func viewDidLoad() {
         super.viewDidLoad()
-
-        // Do any additional setup after loading the view.
+        let currentUser = Auth.auth().currentUser
+        if currentUser != nil {
+            let defaults: UserDefaults = UserDefaults.standard
+            if defaults.bool(forKey: "ISRemember") {
+                rememberMeSwitch.setOn(true, animated: false)
+                print("already logged user")
+            }
+            else {
+                rememberMeSwitch.setOn(false, animated: false)
+            }
+        }
+        rememberMeSwitch.addTarget(self, action: #selector(self.rememberMeSwitchChck(_:)), for: .valueChanged)
+  
+        emailTextFld.layer.cornerRadius = 20
+        passwordTextFld.layer.cornerRadius = 20
+        loginImageView.image = UIImage(named: "chicken")
+        welcomeLbl.text = "Welcome"
+        loginLbl.text =  "Login your account"
+        loginBttn.layer.cornerRadius = 15
+        registerBttn.layer.cornerRadius = 15
     }
+   
+   
+    @IBAction func rememberMeSwitchChck(_ sender: UISwitch) {
+        let defaults: UserDefaults? = UserDefaults.standard
+        if sender.isOn {
+            defaults?.set(true, forKey: "ISRemember")
+            print("switch is on")
+            }
+            else {
+                defaults?.set(false, forKey: "ISRemember")
+                print("switch is off")
+            }
+        }
+    
+    @IBAction func loginBttnClicked(_ sender: Any) {
+        guard let email = emailTextFld.text else { return }
+        guard let password = passwordTextFld.text else { return }
+        if email != "" && password != "" {
+                Auth.auth().signIn(withEmail: email, password: password) { (result, error) in
+                       if error != nil{
+//                           self.errorMessage(titleInput: "Error!", messageInput: error?.localizedDescription ?? "Try again.")
+                           if !email.isValidEmail(email: email){
```

Comment:
I guess email and password check need to be first and then we need to call firebase auth.  Please change order.